Modules/TRD/src/PulseHeight.cxx

```diff
@@ -39,11 +39,11 @@ PulseHeight::~PulseHeight()
 {
 }
 
-void PulseHeight::connectCCDB()
+void PulseHeight::retrieveCCDBSettings()
 {
-  auto& ccdbmgr = o2::ccdb::BasicCCDBManager::instance();
-  //ccdbmgr.setURL("http://localhost:8080");
-  mNoiseMap.reset(ccdbmgr.get<o2::trd::NoiseStatusMCM>("/TRD/Calib/NoiseMapMCM"));
+  //auto& ccdbmgr = o2::ccdb::BasicCCDBManager::instance();
+  //mNoiseMap.reset(ccdbmgr.get<o2::trd::NoiseStatusMCM>("/TRD/Calib/NoiseMapMCM"));
+  mNoiseMap.reset((o2::trd::NoiseStatusMCM*)retrieveCondition("/TRD/Calib/NoiseMapMCM"));
```

Comment:
We forbid c-style casts in O2. Please use `reinterpret_cast`, `static_cast` or `dynamic_cast` depending on the case.